core/src/services/foundationdb/backend.rs

```diff
@@ -25,31 +25,54 @@ use foundationdb::api::NetworkAutoStop;
 use foundationdb::Database;
 
 use crate::raw::adapters::kv;
-use crate::raw::normalize_root;
+use crate::raw::*;
 use crate::Builder;
 use crate::Error;
 use crate::ErrorKind;
 use crate::Scheme;
 use crate::*;
 
+use serde::Deserialize;
+
 /// [foundationdb](https://www.foundationdb.org/) service support.
+///Config for FoundationDB.
+#[derive(Default, Deserialize)]
+#[serde(default)]
+#[non_exhaustive]
+pub struct FoundationConfig {
```

Comment:
Please make sure `FoundationConfig` has been exported in `foundationdb.mod`

Reply:
@Xuanwo Fixed it. It was some error with github on my device that it wasn't pushing the commit I made to mod.rs. It's done. All tests passed.